contentcuration/contentcuration/frontend/channelEdit/vuex/assessmentItem/getters.js

```diff
@@ -29,7 +29,7 @@ export function getAssessmentItemsCount(state) {
  * Consider new assessment items as valid if `ignoreDelayed` is true.
  */
 export function getAssessmentItemsErrors(state) {
-  return function({ contentNodeId, ignoreDelayed = false }) {
+  return function({ contentNodeId, ignoreDelayed = false, modality = null }) {
```

Comment:
From our discussion, I had thought you were going to avoid having to pass in the modality here, and instead read the modality directly from the vuex state using the contentNodeId?